docs/milestone-1-validation.md

```diff
@@ -0,0 +1,99 @@
+# Milestone-1 Validation
```

Comment:
I kinda feel like this file belongs more in citadel than here but can not say for sure.
My reasoning here is that, validating milestones are stepping points in the overall development and may be overwritten or irrelevant by future milestones or releases. Including a doc here feels more permanent. In fact, I would include a warning or note that specifies the document lifecycle is really only related to one milestone.

Reply:
Actually this document is accompanying the other milestone-1 document on GIT, https://github.com/scality/vmware-ose-scality/blob/development/1.0/docs/milestone-1-requirements.md

Reply:
Hmm that sounds like a good idea. As the other file was created by Terry I did not think of renaming it and adding these details. Please let me know if you still believe it is better to have all details in same file

Reply:
Yea I would do that, but I would still approve the PR even if you chose not to.

Reply:
Completed this in e3ae684ae588760d0b37f12fe614eda2c282a644